lib/catalog/martyrology.ts

```diff
@@ -3796,6 +3909,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Theotonius of Coimbra',
       titles: [Titles.Priest],
     },
+    therese_couderc_virgin: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Thérèse Couderc',
+      titles: [Titles.Virgin],
```

Comment:
Add first name.

Is she really celebrated as _virgin_? According to [Wikipedia](https://en.wikipedia.org/wiki/Th%C3%A9r%C3%A8se_Couderc), she was a co-founder of Sisters of the Cenacle, therefore she could be considered a religious.

```suggestion
    therese_marie_victoire_couderc_virgin: {
      canonizationLevel: CanonizationLevels.Saint,
      name: 'Thérèse Couderc',
      titles: [Titles.Virgin],
```